cmd/vcert/commands.go

```diff
@@ -145,6 +147,24 @@ var (
 		vcert getpolicy -u https://tpp.example.com -t <TPP access token> -z "<policy folder DN>"
 		vcert getpolicy -k <VaaS API key> -z "<app name>\<CIT alias>"`,
 	}
+
+	commandSshPickup = &cli.Command{
+		Before:    runBeforeCommand,
+		Name:      commandSshPickupName,
+		Flags:     sshPickupFlags,
+		Action:    doCommandSshPickup,
+		Usage:     "To retrieve a SSH Certificate",
+		UsageText: `vcert sshpickup -u https://tpp.example.com -t <TPP access token> --pickup-id <ssh cert DN>"`,
```

Comment:
Trailing double-quote should be removed.

Reply:
fixed on most recent commit